timeline/templates/timeline/timeline.html

```diff
@@ -1,8 +1,9 @@
-{% extends 'projectroles/project_base.html' %}
+{% extends 'projectroles/base.html' %}
```

Comment:
I'm still seeing changes to `timeline.html`. This template should not be modified in this pull request.